thinc/backends/numpy_ops.pyx

```diff
@@ -82,6 +83,9 @@ class NumpyOps(Ops):
         else:
             return self.xp.empty(shape, dtype=dtype)
 
+    def cblas(self) -> CBlas:
+        return CBlas()
```

Comment:
Do we want to create a new instance here every time this is called?

Reply:
I changed this to use a singleton. Unfortunately, it needs to be a global variable, otherwise `NumpyOps` cannot be pickled, since `CBlas` contains an unserializable C pointer.